react/components/admin/pages/List/index.tsx

```diff
@@ -42,6 +42,11 @@ const List: React.FunctionComponent<Props> = ({ categorizedRoutes }) => (
       routes={sortRoutes(categorizedRoutes.multipleProducts)}
       titleId="admin/pages.admin.pages.list.section.productCollections"
     />
+    <SectionSeparator />
```

Comment:
I'd just check if there should be a condition here checking whether `categorizedRoutes.notFoundSection` is empty and conditionally showing the section.